AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -664,6 +664,12 @@ - (BOOL)shouldMeasureWithSizeRange:(ASSizeRange)constrainedSize
     return NO;
   }
   
+  // If the constrainedSize is completely zero-area, then there is no possibility for layout calculations to be successful.
+  // This also avoids the issue of an inset being applied to 0, creating negative frame values.
+  if (constrainedSize.min.width * constrainedSize.min.height == 0.0 && constrainedSize.max.width * constrainedSize.min.width == 0.0) {
```

Comment:
Should we assert this in this place? Is it more a api user problem or ASDK side?